File: android/app/src/main/java/com/smartexpense/android/data/remote/dto/request/UpdateProfileRequestDto.java
package com.smartexpense.android.data.remote.dto.request;

public class UpdateProfileRequestDto {
    private String displayName;
    private String avatarUrl;
    private Double monthlyBudget;
    private String themeColor;

    public UpdateProfileRequestDto(String displayName, String avatarUrl, Double monthlyBudget, String themeColor) {
        this.displayName = displayName;
        this.avatarUrl = avatarUrl;
        this.monthlyBudget = monthlyBudget;
        this.themeColor = themeColor;
    }
}

File: android/app/src/main/kotlin/com/smartexpense/android/data/remote/dto/request/UpdateProfileRequestDto.kt
package com.smartexpense.android.data.remote.dto.request

import com.google.gson.annotations.SerializedName

data class UpdateProfileRequestDto(
    @SerializedName("displayName") val displayName: String?,
    @SerializedName("avatarUrl") val avatarUrl: String?,
    @SerializedName("monthlyBudget") val monthlyBudget: Double?,
    @SerializedName("themeColor") val themeColor: String?
)
